__tests__/integration.test.js

```diff
@@ -175,4 +175,72 @@ describe("GET requests", () => {
         });
     });
   });
+
+  describe("/api/articles/:article_id/comments", () => {
+    it('returns with status code 200 "OK"', () => {
+      return request(app).get("/api/articles/1/comments").expect(200);
+    });
+
+    it("responds with an array of comment objects", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(Array.isArray(comments)).toBe(true);
+          expect(comments.length).toBeGreaterThan(0);
+
+          for (let i = 0; i < comments.length; i++) {
```

Comment:
I'm not sure this test is needed. It is not proving that the response contains any comments, which the test description is suggesting. I think the next test is more suitable